api/src/main/java/org/hawkular/apm/api/model/analytics/ServiceDeployment.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.api.model.analytics;
+
+import java.util.Collections;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Represents a service deployment, including a list of build stamps.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+public class ServiceDeployment {
+    @JsonInclude
+    private final String name;
+
+    @JsonInclude
+    private final List<BuildStamp> buildStamps;
+
+    /**
+     * Creates a new Service Deployment, based on the given name and list of build stamps.
+     * @param name        the service name. Required.
+     * @param buildStamps the list of build stamps. Optional.
+     */
+    public ServiceDeployment(String name, List<BuildStamp> buildStamps) {
+        if (null == name || name.isEmpty()) {
+            throw new IllegalStateException("The service name cannot be null nor empty.");
+        }
+
+        if (null == buildStamps) {
+            buildStamps = Collections.emptyList();
+        }
+
+        this.name = name;
+        this.buildStamps = buildStamps;
+    }
+
+    /**
+     * The service name.
+     * @return the service name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * A list of build stamps. This list might not be comprehensive and might contain only a limited number of items.
+     * @return a list of build stamps. If no build stamps exist for this service name, an empty list is returned.
+     */
+    public List<BuildStamp> getBuildStamps() {
```

Comment:
Is there any guarantee about time ordering for the build stamps? Not sure whether it should or not - could just be an indication of available versions, and then time order be determined based on the details information returned.